SPECS/metal3-baremetal-operator/metal3-baremetal-operator.spec

```diff
@@ -0,0 +1,69 @@
+%define git_commit c05f5f731a679fd8820bad4804ed04f17619de7c
+Summary:        Bare Metal Operator implements a Kubernetes API for managing bare metal hosts
+Name:           metal3-baremetal-operator
+Version:        0.4.0
+Release:        1%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Libraries
+URL:            https://github.com/metal3-io/baremetal-operator
+#Source0:       https://github.com/metal3-io/baremetal-operator/archive/refs/tags/capm3-v%%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+# The source is the upstream tarball with the vendor dir (created using
+# "go mod vendor") include for offline builds and its top-level directory
+# renamed to %%{name}-%%{version}. It is then repackaged as a tar using the
+# following command:
+#
+#   tar \
+#     --mtime="1970-01-01 00:00Z" \
+#     --sort=name \
+#     --owner=0 \
+#     --group=0 \
+#     --numeric-owner \
+#     --pax-option=exthdr.name=%d/PaxHeaders/%f,delete=atime,delete=ctime \
+#     -cf %%{name}-%%{version}.tar.gz \
+#     %%{name}-%%{version}
+#
+# This command creates a tar with the same cryptographic hash regardless of time
+# or environment. See https://reproducible-builds.org/docs/archives/. It can be
+# recreated using ./generate-sources.sh.
+
+BuildRequires:  golang >= 1.14
```

Comment:
according to github there seems to be at go 1.17 (that said I have not checked for version 0.04 but for their main)